src/Skills.jsx

```diff
@@ -210,8 +211,13 @@ class Skills extends Component {
     let reviews = null;
     if (this.state.reviews){
       reviews = this.state.reviews.map(review => {
-        let header = review.team_member_name + " on " + new Date(review.timestamp).toDateString();
-        return (<Panel header={header} key={review.id}>{review.body}</Panel>);
+        return <ReviewPanel
+          timestamp = {review.timestamp}
+          good = {review.positive}
+          reviewer = {review.team_member_name}
+          >
+            {review.body}
+          </ReviewPanel>
```

Comment:
You could just pass `review` in a single object instead of having multiple props (since they're all contained within `review` anyways, and you can check the shape of `review` with `PropTypes`